### DISCUSSION TITLE
Connecting a Canon EOS 1100d camera to Windows 10.

### DISCUSSION THREAD

**Discussion by Artcool43** [photobooth-app/photobooth-app#195] [Q&A] [answered]
Good afternoon. Help me please. This is the second week I have not been able to get a picture from the Canon EOS 1100d camera. I did everything according to the instructions. The web server is enabled in the digiCamControl application. I'm checking the webserver address. The camera video is displayed in the browser. In Camera Backend Config (http://localhost:8000/#/admin/config/backends) I entered the web server address. I saved everything. But there is no preview from the camera on the main screen and the camera does not react in any way.
I hope for your help. Thank you
![1](https://github.com/photobooth-app/photobooth-app/assets/142512219/d6b5233b-1b74-48f7-a65e-2bac4a3e3f15)
![2](https://github.com/photobooth-app/photobooth-app/assets/142512219/4bbf1db7-a160-45b3-851a-d09b09cd8103)
![3](https://github.com/photobooth-app/photobooth-app/assets/142512219/e6f12560-6b14-4813-bbd1-33b3b2fcca9c)
![4](https://github.com/photobooth-app/photobooth-app/assets/142512219/e5f70bdf-1ae3-4dcc-8682-4ce0fe11a9b4)
![5](https://github.com/photobooth-app/photobooth-app/assets/142512219/3cc0a8fa-7a3a-4e3d-925d-507cce70cd98)

**Comment by mgineer85** [comment, marked as answer]
Hey, in 3rd image please use default url, not the live url. The url is used for control. 
please provide also your config and latest logfile.

**Reply by mgineer85** [reply]
Does the camera work well in digicamcontrol itself including taking pics?

**Reply by Artcool43** [reply]
Good afternoon. Thanks for the answer. I set the default values and everything worked.

**Reply by mgineer85** [reply]
Great, thank you for the feedback.

**Reply by Artcool43** [reply]
Another question in passing. Is it possible to make it in the settings so that the name of the original files is not image_show_20240327-102212-494414.jpg, but as they come from the camera DSC_0013.jpg.

**Reply by mgineer85** [reply]
Currently not implemented. It’s like this so we can treat all files in same fashion from different backends. Maybe you could use exif information for your need?

**Reply by Artcool43** [reply]
Hello. There is one more question. My Canon 1100D camera is mounted vertically (Figure 6). In digiCamControl in live view there are all the settings for camera rotation. And in the preview the position is displayed correctly (Figure 7).
But in photobooth-app (http://localhost:8000/#/), the orientation remains horizontal and the broadcast is rotated 90 degrees. The same is true in the gallery itself (Figure 8). Tell me, where in the settings of the photobooth-app itself can I change the camera orientation? Thank you.
![6](https://github.com/photobooth-app/photobooth-app/assets/142512219/9c358b03-fe9f-4923-9d1d-eda7a4dd001a)
![7](https://github.com/photobooth-app/photobooth-app/assets/142512219/be6218a7-a6b8-449e-946b-84366c1e48fd)
![8](https://github.com/photobooth-app/photobooth-app/assets/142512219/81a8c8e5-27e5-403e-83ef-29b932963e2c)
